### DISCUSSION TITLE
How to fix @layer with talwind Stylelint error

### DISCUSSION THREAD

**Discussion by aradradev** [tailwindlabs/tailwindcss#11719] [Help]
Hello Community, I just started using Tailwind, it is my first time and I've got some issues with the stylelint in my React project. I hope to get help from you guys.

Here are the errors that I am getting when running to fix stylelint error:
7:1  ✖  Unexpected unknown at-rule "@layer"   scss/at-rule-no-unknown
 7:1  ✖  Unexpected unknown at-rule "@layer"   at-rule-no-unknown

**Comment by wongjn** [comment]
It seems like you are using the [`stylelint-scss`](https://www.npmjs.com/package/stylelint-scss) package? If so, as per their recommended settings:

```json
"rules": {
  "at-rule-no-unknown": null,
  "scss/at-rule-no-unknown": true,
}
```
This disables the `at-rule-no-unknown` rule as this will conflict with scss-specific at-rules.

Furthermore, it seems like you may be using an outdated version of `stylelint` and/or `stylelint-scss` packages, as [`@layer` was added to the list of known at-rules in `v14.8.3`](https://github.com/stylelint/stylelint/blob/15.10.2/CHANGELOG.md#1483). You could consider updating `stylelint` and/or `stylelint-scss`. Otherwise, you could consider using [the `ignoreAtRules` configuration option](https://github.com/stylelint-scss/stylelint-scss/tree/v5.0.1/src/rules/at-rule-no-unknown#ignoreatrules-regex-string) to ignore the error for `@layer`.

**Reply by aradradev** [reply]
Thanks @wongjn for your precious resources, It is working now with no linter error. I added just in the stylelintrc.json file layer in the ignoreAtRules and installed this command: npm install stylelint-config-tailwindcss --save-dev
